web/styles/dark_theme.css

```diff
@@ -1081,7 +1081,8 @@
 
     /* Search highlight used in both topics and rendered_markdown */
     .highlight {
-        background-color: hsl(51deg 100% 23%);
+        background-color: hsl(51deg 100% 72%);
```

Comment:
Let's get all of these colors into CSS variables in `zulip.css`. Your PR is a great opportunity to just completely remove these lines from `dark_theme.css` entirely.

Reply:
And especially once this removal is in place, make the `.highlight`-related work its own commit.

Reply:
sure,I will introduce CSS variables for the colors in zulip.css. Moreover, I'll create a separate commit for the .highlight-related changes. can you please elaborate on what do you mean by completely remove these lines